wp-cache-phase2.php

```diff
@@ -1156,8 +1179,11 @@ function wp_cache_shutdown_callback() {
 		// correct Content-Type header for feeds, if we didn't see
 		// it in the response headers already. -- dougal
 		if ( isset( $wp_super_cache_query[ 'is_feed' ] ) ) {
-			$type = get_query_var('feed');
-			$type = str_replace('/','',$type);
+			if ( $type = get_query_var('feed') ) {
```

Comment:
Assigning a value inside an if() is something I'd like to avoid just because it makes the code slightly harder to read. Can you move that back out please?